clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1078,6 +1077,18 @@ public static synchronized void registerCustomInAppTemplates(TemplateProducer pr
         TemplatesManager.register(producer);
     }
 
+    /**
+     * Retrieve a {@link CustomTemplateContext} for a template that is currently displaying. If the provided template
+     * name is not of a currently active template this method returns <code>null</code>.
+     */
+    @Nullable
+    public CustomTemplateContext getActiveContextForTemplate(@NonNull String templateName) {
```

Comment:
would have been nice to have documentation change in PR itself. Can do later now.